src/components/Image.tsx

```diff
@@ -1,7 +1,7 @@
 import { PhotoProvider, PhotoView } from 'react-image-previewer';
 import { CloseButton } from 'react-image-previewer/ui';
 
-const ImageComponent = ({ imgsrc, alt }: { imgsrc: string; alt?: string }) => {
+const Image = ({ imgsrc, alt }: { imgsrc?: string; alt?: string }) => {
```

Comment:
I had to rename because astro implicitly creates components, so there were 2 `image-component` one from `.tsx` another from `.astro`